sourcegit/cli/watch_fedora_ci.py

```diff
@@ -27,10 +20,6 @@ def watcher(message_id):
 
     :return: int, retcode
     """
-    # we can watch for runs directly:
-    # "org.centos.prod.ci.pipeline.allpackages.complete"
-    topic = "org.fedoraproject.prod.pagure.pull-request.flag.added"
-
     a = SourceGitAPI()
```

Comment:
Maybe we can use a better name for this. (And also in `watch_pr`.)

Reply:
we can also tell that we like ugly code and we don't want contributors :D

yeah, I'll fix it